From: =?utf-8?q?=22IOhannes_m_zm=C3=B6lnig_=28Debian/GNU=29=22?=
 <umlaeute@debian.org>
Date: Tue, 9 Oct 2018 20:56:41 +0200
Subject: removed broken pytest config from setup.cfg

---
 setup.cfg | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 0ecced7..7ab330e 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -4,9 +4,6 @@ universal = 1
 [metadata]
 license_file = LICENSE.txt
 
-[tool:pytest]
-addopts = -v --timeout=300 --cov=can --cov-config=setup.cfg
-
 [coverage:run]
 branch = False
 omit = 
